_posts/admin/2017-05-14-keystore-management.md

```diff
@@ -17,9 +17,36 @@ In order to modify the configuration of a specific keystore service instance, th
 
 ![]({{ site.baseurl }}/assets/images/admin/KeystoreConfig2.png)
 
-For the **org.eclipse.kura.core.keystore.KeystoreServiceImpl** factory, the user can customise the following options:
+The following KeystoreService factories are available:
 
-- Keystore Path: identifies the path in the filesystem. The value should reference an existing keystore. The value cannot be empty.
-- Keystore Password: the corresponding keystore password
-- Randomize Password: a boolean flag that allows the user to specify if the keystore password needs to be randomised at the next framework boot. If set true, the framework will try to access the identified keystore and randomise the password. The new password will be persisted in the framework snapshot. Once successfully randomised, the flag will be automatically set to false by the framework.
+## FilesystemKeystoreServiceImpl
+
+The `org.eclipse.kura.core.keystore.FilesystemKeystoreServiceImpl` factory provides a KeystoreService implementation that stores the private keys and certificates as a file, the user can customise the following options:
```

Comment:
`certificates as a file, the user can customise the following options:`

certificates as a file. The user can customise the following options: